pkg/security/probe/probe_ebpf.go

```diff
@@ -149,6 +149,9 @@ type EBPFProbe struct {
 	// On demand
```

Comment:
I think we should also modify other probes, right? (probe_ebpfless.go and even maybe windows?)

Reply:
Windows we can't, but yes maybe ebpfless